packages/block-library/src/button/style.scss

```diff
@@ -49,3 +49,14 @@ $blocks-button__height: 56px;
 		border: 2px solid;
 	}
 }
+
+.editor-format-toolbar__url-inline {
```

Comment:
I think this is copied from my other branch, but unfortunately I was just hacking stuff together there, so we'll need to change a few things to bring it up to the right standard.

First thing is that this should be moved to the `button/editor.scss`, since they're styles that should only appear in the editor.

This should also have a different classname as this form isn't part of the editor toolbar, perhaps something like `wp-block-button__link`.

Reply:
Addressed [here](https://github.com/WordPress/gutenberg/pull/10128/commits/9ef9fdfc02d2da67a7a4be68245bb37bc6138d7d#diff-48774fc728094a4cbd45abc1d467ee70).

I have used `wp-block-button__inline-link` instead because the button `RichText` element already uses the `wp-block-button__link` class.

Thank you ❤️